src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -2230,6 +2230,8 @@ void RadioSpinel<InterfaceType, ProcessContextType>::HandleRcpUnexpectedReset(sp
 template <typename InterfaceType, typename ProcessContextType>
 void RadioSpinel<InterfaceType, ProcessContextType>::HandleRcpTimeout(void)
 {
+    otLogCritPlat("Handle RCP timeout");
```

Comment:
Is it better to add logs where the timeout happened?

Reply:
Updated